packages/tools/replay-tool/src/helpers.ts

```diff
@@ -119,16 +125,36 @@ export async function loadContainer(
         summaryOptions: { disableSummaries: true, maxOpsSinceLastSummary: 100000 },
         gcOptions: { writeDataAtRoot: true },
     };
-    const codeLoader = new ReplayCodeLoader(new ReplayRuntimeFactory(runtimeOptions, dataStoreRegistries));
+    const codeLoader = new ReplayCodeLoader(
+        new ReplayRuntimeFactory(runtimeOptions, dataStoreRegistries, requestHandlers));
 
     // Load the Fluid document
     const loader = new Loader({
         urlResolver,
         documentServiceFactory,
         codeLoader,
-        options: {},
+        options: loaderOptions ?? {},
         logger,
     });
 
     return loader.resolve({ url: resolved.url });
 }
+
+export async function containerRuntimeResolver(request: IRequest, runtime: IContainerRuntime){
+    if (request.url === "/containerRuntime") {
+        return { mimeType: "fluid/object", status: 200, value: runtime };
+    } else {
+        return create404Response(request);
+    }
+};
+
+export async function uploadContainerSummary(container: IContainer) {
```

Comment:
nit: Rename to `uploadSummary`. I don't think the term container adds any value.